hod/options.py

```diff
@@ -64,4 +64,8 @@ def validate_pbs_option(options):
     if not options.workdir:
         _log.error('No workdir ("--workdir") provided')
         return False
+    if not options.hod_module:
```

Comment:
This function is also called for `genconfig`, does it make sense to require `--hod-module` for `genconfig`?